webpack.config.js

```diff
@@ -31,29 +31,25 @@ function buildConfig({
         mode: 'development',
         devtool: 'source-map',
         entry: {
-            // include dist in entry point so that when running dev server,
-            // we can access the files with /dist/...
-            [`dist/transformers${name}`]: './src/transformers.js',
-            [`dist/transformers${name}.min`]: './src/transformers.js',
+            [`transformers${name}`]: './src/transformers.js',
+            [`transformers${name}.min`]: './src/transformers.js',
         },
         output: {
             filename: `[name]${suffix}`,
-            path: __dirname,
+            path: path.join(__dirname, 'dist'),
             library: {
                 type,
             },
+            assetModuleFilename: '[name][ext]',
+            chunkFormat: 'module',
+        },
+        module: {
+            parser: {
+                javascript: {
+                    importMeta: false
+                }
+            }
         },
```

Comment:
This is required to preserve `import.meta.url` in the code. Otherwise webpack will replace the expression to a static string starting with "file://", which breaks worker/wasm loading.

Reply:
thanks!